### PR TITLE
Check status.conditions exists before getting bundle lastUpdateTime - #6771

### LINKED ISSUE #6771: Rancher UI Can't Get to Bundles Page if Bundle Status is does not exist
**Rancher Server Setup**
- Rancher version: Rancher v2.6.4 and Rancher v2.6.5
- Installation option (Docker install/Helm Chart): Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE1

**Information about the Cluster**
- Kubernetes version: v1.21.x
- Cluster Type (Local/Downstream): Local
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? Admin
  - If custom, define the set of permissions:

   
**Describe the bug**

If I delete a bundle and refresh the gitrepo object and the status of the object is not present then the UI fails to allow me to view the bundles page.

**To Reproduce**
 
1. Delete bundle
2. Refresh gitrepo object and verify bundle exists with kubectl
3. Browser log indicates missing properties.

**Result**
I can't view the Bundle page to further debug what went wrong.

**Expected Result**

I expect to be able to view Bundle objects even if they do have the status field.

**Screenshots**

Chrome
image
Safari
image

**Additional context**

### PR DESCRIPTION
**Pull request by Shavindra**
### Summary
Fixes #6771 - Rancher UI Can't Get to Bundles Page if Bundle Status is does not exist #6771

### Occurred changes and/or fixed issues
I couldn't really replicate the issue. I expect the schema and everything to be in place. However, I added optional property check to `this.status.conditions` for getter `lastUpdateTime`

### Areas or cases that should be tested

1. Navigate to Continuous Delivery > Git Repos
2. Click on Add repository
3. Use your own git repo or examples from https://github.com/rancher/fleet-examples to add a repo.
4. Click through newly created repo.
5. Delete all bundles from repo. Leave the repo in place.
6. Refresh gitrepo object and verify bundle exists with kubectl
7. There should not be anymore errors.

### CODE REVIEW

**Review by jordojordo — CHANGES_REQUESTED** [review]
This solves the problem with `status.conditions`, although I'm receiving a different error of `row.status` returning `undefined`. Seems to be stemming from the `v-if` conditional [here](https://github.com/rancher/dashboard/blob/a10327efcd52e29492f82226070e74ef75a92fa7/shell/components/fleet/FleetBundles.vue#L120-L122)

rowstatus

**Comment by Shavindra** [comment]
@jordojordo I am not sure why I still can't replicate the issue. I've update the code anyway.

**Review by jordojordo — CHANGES_REQUESTED** [review]
@Shavindra I was able to replicate the issue by deleting all of the bundles and force updating afterwards.

https://user-images.githubusercontent.com/40806497/189659488-c5952c36-251c-4612-badc-90a15318f07b.mp4

Still getting the `row.status is undefined` but I was able to stop it by changing the `v-else` to `v-else-if="row.status"`
```vue
<span v-else-if="row.status">{{ row.status.summary.desiredReady }}</span>
```

File: shell/components/fleet/FleetBundles.vue [hunk 1]
```diff
@@ -103,9 +103,15 @@ export default {
...
+
+  methods: {
+    displayWarning(row) {
+      return !!row.status?.summary && (row.status.summary.desiredReady !== row.status.summary.ready);
+    }
+  }
 };
 </script>
-
+ß
```
Comment (jordojordo) [inline shell/components/fleet/FleetBundles.vue:114]: This should be removed

**Review by jordojordo — CHANGES_REQUESTED** [review]
One minor change, but lgtm otherwise.

**Review by jordojordo — APPROVED** [review]

**Pull request merged**